modules/apps/data-engine/data-engine-js-components-web/src/main/resources/META-INF/resources/js/custom/form/components/PartialResults.tsx

```diff
@@ -68,39 +82,47 @@ const PartialResults: React.FC<IProps> = ({
 
 				{Liferay.Language.get('back')}
 			</ClayButton>
-
-			<div className="lfr-de__partial-results-entries">
-				<div className="align-items-center">
-					<span className="lfr-de__partial-results-title text-truncate">
-						{totalItems === 1
-							? Liferay.Util.sub(
-									Liferay.Language.get('x-entry'),
-									[totalItems]
-							  )
-							: Liferay.Util.sub(
-									Liferay.Language.get('x-entries'),
-									[totalItems]
-							  )}
-					</span>
-				</div>
-
-				<div className="align-items-center">
-					<span className="lfr-de__partial-results-subtitle text-truncate">
-						{totalItems > 0
-							? lastModifiedDate
-							: Liferay.Language.get('there-are-no-entries')}
-					</span>
-				</div>
-			</div>
-
-			<FormReport
-				data={data}
-				fields={fields}
-				formReportRecordsFieldValuesURL={
-					formReportRecordsFieldValuesURL
-				}
-				portletNamespace={portletNamespace}
-			/>
+			{error ||
+				(!loading && (
```

Comment:
So we could just check a single state here:
```suggestion
			{!loading && (
```